src/main/java/FastCar.java

```diff
@@ -0,0 +1,44 @@
+public class FastCar implements Tradable, Drivable{
+    private int price;
+    private int s;
+    public FastCar(int price, int s){
+        this.price = price;
+        this.s = s;
+    }
+    @Override
+    public int getPrice() {
+        return price;
+    }
+
+    public void setS(int s) {
+        this.s = s;
+    }
+
+    public int getS() {
+        return s;
+    }
+
+    @Override
+    public void downgradeSpeed() {
+        this.s -= 1;
+    }
+
+    @Override
+    public void upgradeSpeed() {
+        this.s += 1;
+    }
+
+    @Override
+    public int getMaxSpeed() {
+        return this.getS();
+    }
+
+    public String drive(){
+        double c = Math.random() * 100;
+        if(c > 50.1){
+            this.s -= 1;
+            return "You have crashed!";
+        }
+        return "arrived to destination";
+    }
```

Comment:
Missing method documentation (i.e. what does the function do, parameters (if any) and return)